ui/src/project.ts

```diff
@@ -384,3 +384,9 @@ export const repositoryPathValidationStore = (
     );
   }
 };
+
+// Checks if the provided user is part of the maintainer list of the project.
+// FIXME(xla): Urns should be properly typed.
+export const isMaintainer = (project: Project, userUrn: string): boolean => {
+  return project.metadata.maintainers.includes(userUrn);
+};
```

Comment:
```suggestion
export const isMaintainer = (userUrn: string, project: Project): boolean => {
  return project.metadata.maintainers.includes(userUrn);
};
```

To make the order of the arguments read better: `isMaintainer(userUrn, project)` <=> "is <user> maintainer of <project>"